client/src/geyser_consumer.rs

```diff
@@ -97,73 +96,6 @@ impl GeyserConsumer {
         Self { client, exit }
     }
 
-    pub async fn consume_account_updates(
```

Comment:
why delete this?

Reply:
not being used anywhere rn so was throwing error. will add back in w/ ok dead code